### DISCUSSION TITLE
Discord asks for phone verification on every attempt to login

### DISCUSSION THREAD

**Discussion by foxsouns** [QuiteAFancyEmerald/InvisiProxy#194] [Q&A]
Is there any way to get discord to work on holyub without this? i'm able to create accounts, but i am unable to login. i've been using womginx, tried both private and classic. ~i dont think corrosion works with discord, at least not with my attempts~, and pydodge doesnt seem to work at all on the instance i have.

edit: corrosion works, but still getting the phone verification request.

**Comment by nanospearing** [comment]
This is very common and Un fixable at the current moment. Sorry.

**Reply by foxsouns** [reply]
unfortunate. is it a for sure thing, or is it just with certain holyub mirrors?

**Reply by DevilGlitch** [reply]
I figured out a way

**Reply by DevilGlitch** [reply]
By using your phone to sign in using QR code or verifying on another computer it allows you to verify

**Reply by DevilGlitch** [reply]
@BuggyThe1st && @foxsouns && @KozmikNano

**Reply by DevilGlitch** [reply]
hopefully this works for you sorry if it doesn't

**Reply by ghost** [reply]
Alright thanks

**Reply by foxsouns** [reply]
> By using your phone to sign in using QR code or verifying on another computer it allows you to verify

meaning if you dont have another device, you cant use discord on HU
sucks

**Comment by nanospearing** [comment]
The reason this happens is because many people are on the site with the same ip being forwarded to discord. When this happens they think that it is a bot trying to connect. The only way that I can think of would to have a prove site, but even then it may not work.

**Reply by foxsouns** [reply]
ah, i see. i'm going to try to self host and see if i can get it to work there.

**Comment by ghost** [comment]
I am having the same problem!
Can you ping me when you find a feasible solution?
